phpcs.xml

```diff
@@ -1,54 +1,26 @@
-<?xml version="1.0" ?>
+<?xml version="1.0" encoding="UTF-8"?>
 <!--
   Based on Examples for Developers phpcs.xml.dist. Modified for use for Drupal
   projects. See https://git.drupalcode.org/project/examples/blob/HEAD/phpcs.xml.dist
   for source.
 -->
 <ruleset name="AxelerantDrupal">
   <description>Axelerant baseline rules for PHP CodeSniffer</description>
-
-  <!--Exclude folders used by common frontend tools.-->
-  <exclude-pattern>*/bower_components/*</exclude-pattern>
-  <exclude-pattern>*/node_modules/*</exclude-pattern>
-  <!--Exclude third party code.-->
-  <exclude-pattern>./assets/vendor/*</exclude-pattern>
-
   <file>.</file>
-
-  <!-- PHPCS Arguments -->
-  <arg name="extensions" value="inc,yml,install,module,php,profile,test,theme" />
-  <arg name="colors" />
-
-  <!--Allow global variables in settings file.-->
-  <rule ref="DrupalPractice.CodeAnalysis.VariableAnalysis.UndefinedVariable">
-    <exclude-pattern>settings(\.[a-zA-Z0-9]+)?\.php</exclude-pattern>
```

Comment:
If we remove this, phpcs will stop catching UndefinedVariable. Currently it did not catch it in settings.php because `/web/sites/default` is excluded in grumphp. So if don't want that exclude pattern, we should only keep the rule without any exclude as settings file are already excluded in grumphp. 

`<rule ref="DrupalPractice.CodeAnalysis.VariableAnalysis.UndefinedVariable" />`

Same applies for other `DrupalPractice.Commenting` rules.